github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -129,6 +130,67 @@ periodics:
         cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
         ./hack/bump-centos-version.sh &&
         GIT_ASKPASS=../project-infra/hack/git-askpass.sh ../project-infra/hack/git-pr.sh -c "PHASES=linux BYPASS_PMAN_CHANGE_CHECK=true ./publish.sh" -r kubevirtci -b bump-centos-stream -T main -p $(pwd) -s "Automatic bump of CentOS Stream to latest"
+        # For passing centos image tag to dependent (s390x) prow job
+        SHORT_SHA=$(git rev-parse --short HEAD) &&
+        image_tag=$(cat cluster-provision/k8s/base-image | cut -d ':' -f 2) &&
+        echo "$image_tag" > amd64-centos9-$SHORT_SHA &&
+        gsutil cp ./amd64-centos9-$SHORT_SHA gs://kubevirt-prow/release/kubevirt/kubevirtci/amd64-centos9-$SHORT_SHA
+      # docker-in-docker needs privileged mode
+      env:
+      - name: GIMME_GO_VERSION
+        value: "1.22.5"
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          memory: "29Gi"
+- name: periodic-kubevirtci-bump-centos-base-s390x
+  cron: "0 4 * * 2"
+  annotations:
+    testgrid-create-test-group: "false"
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  max_concurrency: 1
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirtci
+    base_ref: main
+    workdir: true
+  labels:
+    preset-docker-mirror-proxy: "true"
```

Comment:
You need to remove this preset as the s390x cluster does not have a  docker mirror proxy configured.

Reply:
Sure, will do